pilot/pkg/networking/core/listener_waypoint.go

```diff
@@ -365,7 +390,8 @@ func (lb *ListenerBuilder) buildWaypointInternal(wls []model.WorkloadInfo, svcs
 		}
 	}
 
-	{
+	// TODO: is this correct?
```

Comment:
At least for now I think so

Reply:
Thanks for the confirmation; it'll probably change when we add workload addressing